### PR TITLE
Search refactor alex Close #37

### LINKED ISSUE #37: Refactor search code to use fields suggested by the client
DOD: when the advanced search has only the fields the client requested: Position, High School, Graduation Year, Club Coach, High School Coach

### PR DESCRIPTION
**Pull request by AlexKSparks**
Athlete Profile Model: added AAUCoach
SearchViewModel / SearchResultsViewModel: Changed attributes to match new search
AdvancedSearchView / SearchResultsView: Changed to search athlete attributes listed in DOD
SearchController: Edited to take in correct parameters from changed ViewModels

### CODE REVIEW

**Review by keithrutler — APPROVED** [review]
The search results view model is so thin now. I'm ok with this pull, it has the changes that would let me start that search parameter issue I brought over from from sprint 1

**Review by Daniel-Watkins — CHANGES_REQUESTED** [review]
I thought you were also refactoring the Basic Search post method based on our discussion on 5/1. Without this completed the search bar at the top of the nav bar will not function correctly.

**Comment by AlexKSparks** [comment]
Sorry, I forgot to include that. New commit with those changes has been submitted Daniel. Thanks for the reminder

File: gybitg/Controllers/SearchController.cs [hunk 1]
```diff
@@ -89,79 +85,38 @@ public async Task<IActionResult> SearchResults(SearchViewModel SearchParam)
...
-            decimal SearchMaxMPG = SearchParam.MaxMPG;
-            decimal SearchMinMPG = SearchParam.MinMPG;
-
-            decimal SearchMaxTPMG = SearchParam.MaxTPMG;
-            decimal SearchMinTPMG = SearchParam.MinTPMG;
-
-            decimal SearchMaxFTMG = SearchParam.MaxFTMG;
-            decimal SearchMinFTMG = SearchParam.MinFTMG;
+            string SearchHS = SearchParam.HighSchool;
+            string SearchAAU = SearchParam.AAUId;
```
Comment (adamkohler) [inline gybitg/Controllers/SearchController.cs:89]: I had not added info for AAUId for the repo athletes, I will do that asap if that is something that will be searchable.  Sorry.

**Review by adamkohler — APPROVED** [review]
Looks good, I had forgotten about the AAUId, I will get that data into each repo athlete along with the AAUCoach and be ready to merge again once you do yours.

File: gybitg/Controllers/SearchController.cs (hunk 1 above)
Comment (AlexKSparks) [reply gybitg/Controllers/SearchController.cs:89]: Sounds good

**Review by Daniel-Watkins — APPROVED** [review]
Looks fine to merge now.

**Review by ksdurg — APPROVED** [review]
That was a lot of work.  Great job, Alex!

**Pull request merged**